doc/plugins/limit-req.md

```diff
@@ -26,8 +26,8 @@
   - [Attributes](#attributes)
   - [How To Enable](#how-to-enable)
   - [Test Plugin](#test-plugin)
-    - [How to enable on the `consumer`](#how-to-enable-on-the-consumer)
-    - [Test Plugin](#test-plugin-1)
+  - [How to enable on the `consumer`](#how-to-enable-on-the-consumer)
+  - [Test Plugin](#test-plugin-1)
```

Comment:
![image](https://user-images.githubusercontent.com/6814606/93855243-c4597d00-fce9-11ea-98a0-96b768e13ed2.png)

@Firstsawyou

Reply:
I think `Test Plugin` is a part of "How to enable on the `consumer`".

Reply:
fixed